contentcuration/contentcuration/frontend/channelEdit/views/ImportFromChannels/SearchOrBrowseWindow.vue

```diff
@@ -151,6 +157,7 @@
             },
             query: {
               ...this.$route.query,
+              ...(this.languages !== '' ? { languages: this.languages } : ''),
```

Comment:
This is resulting in a bug -- you can't set two languages together in import search. 

Change the logic such that when we are on `isBrowsing` page then only apply that language.

Reply:
@vkWeb I didn't understand correctly. Could you clarify a bit more?

Reply:
Try doing this -- go to import channel search, enter a search term and press enter, then on the results page, try applying two language filters. You will not be able to apply two language filters. That's the bug. 

On the results page's language filter, we must be able to apply multiple languages, but with the code change that you did, now, we are not able to do it. 

Was I able to explain clearly or is it blurry? Please let me know.

Reply:
> Try doing this -- go to import channel search, enter a search term and press enter, then on the results page, try applying two language filters. You will not be able to apply two language filters. That's the bug.
> 
> On the results page's language filter, we must be able to apply multiple languages, but with the code change that you did, now, we are not able to do it.
> 
> Was I able to explain clearly or is it blurry? Please let me know.


https://github.com/learningequality/studio/assets/84982038/c69f9b1c-24e1-4175-b731-3bcb59d4d0f3

@vkWeb Is this what you were trying to do? I'm able to apply multiple languages.

Reply:
The below video shows the bug, do turn on the sound to listen my voice :D

[video-2024-01-10_19.25.30.webm](https://github.com/learningequality/studio/assets/26724128/50d36929-e50b-4072-ae0f-4bf73c506564)

Reply:
Thanks, @vkWeb got it!

Reply:
@vkWeb I tried the same, and it's not causing any error on my device. I have pushed the changes for the previous bugs, it got resolved on its own. Could you check it out?


https://github.com/learningequality/studio/assets/84982038/7b73607c-1499-4d49-b731-9d97d3a9ac9b

Reply:
It got resolved because you added `this.languages = ' '` in handleSearchTerm.